### PR TITLE
Configure Index Prefix on Settings Page

### PR DESCRIPTION
**Pull request by allan23**
Allow index prefix to be configured on settings page instead of just within wp-config.php.

### CODE REVIEW

**Comment by allan23** [comment]
Using JavaScript, verify that the endpoint is ElasticPress.io. If so, then show the prefix field.

**Comment by allan23** [comment]
@tott and @oscarssanchez, can you please review?

File: classes/class-ep-config.php [hunk 1]
```diff
@@ -88,6 +113,51 @@ public function get_host() {
...
+
+		if ( defined( 'EP_IS_NETWORK' ) && EP_IS_NETWORK && ep_is_epio() ) {
+			$credentials = ep_sanitize_credentials( get_site_option( 'ep_credentials', false ) );
+		} elseif ( ep_is_epio() ) {
+			$credentials = ep_sanitize_credentials( get_option( 'ep_credentials', false ) );
+		} else {
+			$credentials = [];
+		}
+
+		if ( ! is_array( $credentials ) ) {
```
Comment (brandwaffle) [inline classes/class-ep-config.php]: @allan23 if line 151 is executed then this section will never set the blank array keys. Is this intended behavior?

Comment (allan23) [reply classes/class-ep-config.php]: It is not. Great catch!

**Comment by allan23** [comment]
@brandwaffle PR updated

**Pull request merged**